libs/common/src/vault/utils.ts

```diff
@@ -65,10 +70,12 @@ export function isCardExpired(cipherCard: CardView): boolean {
 
     if (normalizedYear && expMonth) {
       // `Date` months are zero-indexed
-      const parsedMonth =
-        parseInt(expMonth) - 1 ||
-        // Add a month floor of 0 to protect against an invalid low month value of "0"
-        0;
+      let parsedMonth = parseInt(expMonth) - 1;
+
+      // Add a month floor of 0 to protect against an invalid low month value of "0" or negative integers
+      if (parsedMonth < 1) {
+        parsedMonth = 0;
+      }
```

Comment:
🎨 - You could consider using `Math.max` here

```suggestion
      // Add a month floor of 0 to protect against an invalid low month value of "0" or negative integers
      const parsedMonth = Math.max(parseInt(expMonth) - 1, 0);
```

Reply:
yep - I also need to cover `expMonth` being parsed to `NaN`

Reply:
6d0273867d7c13c99098535897d6ebba3c24d281